spark-cassandra-connector/src/it/scala/org/apache/spark/sql/cassandra/CassandraSQLClusterLevelSpec.scala

```diff
@@ -51,12 +50,22 @@ class CassandraSQLClusterLevelSpec extends SparkCassandraITFlatSpecBase {
       .set("spark.cassandra.connection.host", getHost(1).getHostAddress)
       .set("spark.cassandra.connection.native.port", getNativePort(1).toString)
       .set("spark.cassandra.connection.rpc.port", getRpcPort(1).toString)
-    cc.addClusterLevelCassandraConnConf("cluster1", conf1)
-      .addClusterLevelCassandraConnConf("cluster2", conf2)
-      .addClusterLevelReadConf("cluster1", sc.getConf)
-      .addClusterLevelWriteConf("cluster1", sc.getConf)
-      .addClusterLevelReadConf("cluster2", sc.getConf)
-      .addClusterLevelWriteConf("cluster2", sc.getConf)
+    cc.addCassandraConnConf(CassandraConnectorConf(conf1), "cluster1")
+    cc.addCassandraConnConf(CassandraConnectorConf(conf2), "cluster2")
+    cc.addClusterLevelReadConf(ReadConf.fromSparkConf(sc.getConf), "cluster1")
+    cc.addClusterLevelWriteConf(WriteConf.fromSparkConf(sc.getConf), "cluster1")
+    cc.addClusterLevelReadConf(ReadConf.fromSparkConf(sc.getConf), "cluster2")
+    cc.addClusterLevelWriteConf(WriteConf.fromSparkConf(sc.getConf), "cluster2")
+  }
+
+  override def afterAll() {
+    super.afterAll()
+    conn.withSessionDo { session =>
+      session.execute("DROP KEYSPACE sql_test1")
+    }
+    conn2.withSessionDo { session =>
+      session.execute("DROP KEYSPACE sql_test2")
+    }
```

Comment:
You'd better use a different keyspace in every test suite, rather than relying on dropping/recreating them. In the future we will want to parallelize integration tests, because they are already taking longer and longer with every release. Fortunately this is hardly as bad as in DSE, but we're heading in that direction. If every test suite used a different keyspace that would be generally safer.